src/main/java/com/metamx/common/StringUtils.java

```diff
@@ -84,4 +84,24 @@ public static String safeFormat(String message, Object... formatArgs)
       return bob.toString();
     }
   }
+
+  // should be used only for estimation.. does not check validity of format
```

Comment:
it would also be good to document in the cases in which it is not equal to `StringUtils.toUtf8(value).length`. Does it tend to under-estimate or does it tend to over-estimate?